RULES_DESCRIPTIONS.md

```diff
@@ -737,7 +737,12 @@ _Configuration_: N/A
 
 _Description_: By convention, receiver names in a method should reflect their identity. For example, if the receiver is of type `Parts`, `p` is an adequate name for it. Contrary to other languages, it is not idiomatic to name receivers as `this` or `self`.
 
-_Configuration_: N/A
+_Configuration_: (optional) a string indicating the maximum length of the receiver name. If not set, name length is not checked.
+
+```toml
+[rule.receiver-naming]
+    arguments=["max-length=2"]
```

Comment:
Why did you decide to have this way of configuration? This seems to be different comparing g to what we have in the other rules. We either have ints directly or we have named args as in unchecked-type-assertion.